### DISCUSSION TITLE
如何在 Tree 中加入 border?

### DISCUSSION THREAD

**Discussion by janlin002** [ant-design/ant-design#42923] [Q&A] [answered]
![截圖 2023-06-08 下午1 47 52](https://github.com/ant-design/ant-design/assets/68535430/ce0c273c-a3fe-40b4-bfaf-11f6d1edadf3)
项目中有个需求是希望可以将Tree里面的内容透过 border(或是任何方式)做区隔，想请问一下，该如何实现？

**Comment by linxianxi** [comment]
这样？https://codesandbox.io/s/ji-ben-antd-5-6-0-forked-pmq1fy?file=/demo.tsx

**Comment by janlin002** [comment]
對，不過可以讓下拉的箭頭顯示在右邊，並且是同一層用一個border，而不是一個options用一個border嗎？

像是這樣:
![截圖 2023-06-08 下午4 26 20](https://github.com/ant-design/ant-design/assets/68535430/de027ba3-f1ad-4b41-8ecd-5530117f47b1)

**Reply by linxianxi** [reply, marked as answer]
箭头位置改变不了。用 Collapse 自己写一个吧

**Reply by janlin002** [reply]
感謝～～我再研究看看
